src/narmvm.rs

```diff
@@ -319,7 +319,8 @@ impl NarmVM{
                 },
                 //0100_0011_01xx_xyyy MUL T1 flags
                 0b0100_0011_0100_0000 => {
-                    self.sreg[reg2] = valuen * valuem;
+                    let (result, _) = (valuen as u32).overflowing_mul(valuem as u32);
```

Comment:
To be fair, this wouldn't actually panic in release mode, but always a good practice to explicitly use `overflowing_XXX` when dealing with anything that can overflow/underflow. I had forgotten here

Reply:
Right, read a few days ago and then promptly forgot about the only-in-testing overflow checks. But best practice is best practice!